NOTICE

```diff
@@ -118,3 +118,14 @@ taverna-ui/src/main/java/org/apache/taverna/lang/ui/treetable/
   for use in the design, construction, operation or maintenance of any
   nuclear facility.
 
+---------------------------------------------------------
+incubator-taverna-workbench/taverna-perspective-biocatalogue/src/main/resources/net/sf/taverna/t2/ui/perspectives/biocatalogue/service_icons/soap_rest_multitype_unknown.pdn
+
+TODO: License should be clarified
+
+---------------------------------------------------------
+incubator-taverna-workbench/taverna-perspective-biocatalogue/src/main/resources/net/sf/taverna/t2/ui/perspectives/biocatalogue/famfamfam_silk/add - tick.pdn
```

Comment:
I think the whole famfamfam_silk folder are from
http://www.famfamfam.com/lab/icons/silk/ - perhaps the easiest is to
download from famfamfam.com and compare some of the PNGs.


I think the other icons we have made.. particularly very specific ones
like "soap_rest".. but good to check.


git log --follow
taverna-perspective-biocatalogue/src/main/resources/net/sf/taverna/t2/ui/perspectives/biocatalogue/service_icons/soap_rest_multitype_unknown.pdn

starts with

commit 06d873660215e91262cc46661202441f2f140927
Author: alan@mygrid.org.uk
<alan@mygrid.org.uk@bf327186-88b3-11dd-a302-d386e5130c1c>
Date:   Fri Nov 5 17:51:24 2010 +0000

    Initial copy from mygrid labs as of 2010-11-05

    git-svn-id:
https://taverna.googlecode.com/svn/taverna/ui/net.sf.taverna.t2.ui-exts/branches/maintenance@11827
bf327186-88b3-11dd-a302-d386e5130c1c


This originates from what used to be
http://mygrid-labs.googlecode.com/svn/ - luckily archived at
https://github.com/myGrid/mygrid-labs after Google Code closed.

"Find File" in here finds the soap_rest_multitype_unknown.pdn file -
however its "history" is a bit short - probably because it's svn
translated to git history.. so it's not obvious where it's copied
"from".

After a bit of digging of files available in the parent commits I find:

https://github.com/myGrid/mygrid-labs/commit/9fa567a59990435822314dc35644395eba049740

where sergejs@mygrid.org.uk in  12 Apr 2010 (an intern at Univ of
Manchester) does an "initial commit" of the biocatalogue perspective,
including
lots of icons in
https://github.com/myGrid/mygrid-labs/tree/9fa567a59990435822314dc35644395eba049740/biocatalogue-perspective/trunk/src/main/resources/net/sf/taverna/t2/ui/perspectives/biocatalogue

now from this we don't have a clear history trail.. as that includes
those fam_fam icons. :-(

(So folks - always have better commit messages than a massive "initial
commit"  - and always say where externally sourced files come from!)


I think for most of these they might have just been "from the
internet" :-/  -- and so we would need to remove those that by their
looks don't obviously are not made by this project (e.g.
biocatalogue.png would be our) - and do filename or image searches on
the rest...  (or just delete them).

ajax-loader.gif is probably from http://ajaxload.info/#preview - which
says  (close your eyes)

 >Generated gifs can be used, modified and distributed under the terms
of the â€‹Do What The Fuck You Want To Public License.



On 13 September 2016 at 02:55, M.G.T.R Manamgoda
<notifications@github.com> wrote:
> In NOTICE:
>
>> @@ -118,3 +118,14 @@
>> taverna-ui/src/main/java/org/apache/taverna/lang/ui/treetable/
>>    for use in the design, construction, operation or maintenance of any
>>    nuclear facility.
>>
>> +---------------------------------------------------------
>>
>> +incubator-taverna-workbench/taverna-perspective-biocatalogue/src/main/resources/net/sf/taverna/t2/ui/perspectives/biocatalogue/service_icons/soap_rest_multitype_unknown.pdn
>> +
>> +TODO: License should be clarified
>> +
>> +---------------------------------------------------------
>>
>> +incubator-taverna-workbench/taverna-perspective-biocatalogue/src/main/resources/net/sf/taverna/t2/ui/perspectives/biocatalogue/famfamfam_silk/add
>> - tick.pdn
>
> How can we find, there is no document mentioning under which license those
> Icons are ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.



-- 
Stian Soiland-Reyes
http://orcid.org/0000-0001-9842-9718